platform_umbrella/apps/common_core/lib/common_core/resources/sso/keycloak.ex

```diff
@@ -65,6 +65,7 @@ defmodule CommonCore.Resources.Keycloak do
     |> B.name("keycloak")
     |> B.namespace(namespace)
     |> B.data(data)
+    |> F.require(battery.config.admin_password != "")
```

Comment:
Belt and suspenders. Just prevents secret from being created ( and keycloak from starting) if there's not an admin password.